src/item.cpp

```diff
@@ -4157,7 +4157,7 @@ void item::final_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
     contents_info( info, parts, batch, debug );
 
     if( get_option<bool>( "ENABLE_ASCII_ART" ) ) {
-        const ascii_art_id art = type->picture_id;
+        const ascii_art_id art( type->get_id().str() );
```

Comment:
Like said in the dev discord this seem very hacky. It might be better to add a `item_id` entry to the ascii art object and then check for that once everything is loaded.

Reply:
I tried to do that, the same method that was used to make the former ascii_picture variable from the json factory, but it really hated that it was coming from "id" , I had the same problem over with monster.cpp

Reply:
Unless you mean something else? as the ascii art objects already have an id

Reply:
I think you can make "id" of ascii art definitions an array, and change the c++ code so the ids in the array all point to the same ascii art object.

Reply:
Yes that is something I want to do, but I need to find an example as I am not sure how to do it

Reply:
"migration", "talk_topic", and "overmap_terrain" objects support using an array for the object id. (You can find them by searching for `"id": [` in the json folder)